R/actions.R

```diff
@@ -373,7 +373,12 @@ input_val_handler <- function(val, binding) {
     handler <- `%:::%`("shiny", "inputHandlers")$get(binding)
   }
   if (!is.null(handler)) {
-    return(handler(val))
+    val <- handler(val)
+    if ("air.datetime" %in% binding) {
+      if (is.null(val)) return(c(Inf, -Inf))
+      if (length(val) == 1) return(c(val, Inf))
```

Comment:
Are you sure it shouldn't be `c(val, val)`?

Reply:
hmm, this is for single date in gui. I made the assumption that when a single date is chosen in date picker we use this as start date.

Maybe it could be cleaner if we use this as single point, but it will not give us a lot of value as in date time it's hard to select a single value with precision. You can define two values.

The feedback plot is also giving nice outcome that we filter from the start date to the end. 

I personally prefer to use this as start date point.

Reply:
As discussed, changed to val, val.